geth/jail/extensions/extensions.go

```diff
@@ -0,0 +1,39 @@
+package extensions
+
+import (
+	"github.com/robertkrimen/otto"
+)
+
+// ExtensionFunction types a function type which is used to define
+// given extensions to be registered.
+type ExtensionFunction func(*otto.Otto) error
+
+// exts defines a package level variable to hold registered otto extensions
+// to allow us jail break normal otto.VM API or behaviours as necessary.
```

Comment:
there is no "jail break", our Jailed environment is "jailed" because we execute JS inside of Otto (as opposed to executing JS on mobile side inside browser). So, having this as "exts defines a package level variable to hold registered otto extensions" i.e. w/o the second line comment is enough.

P.S. Overall look for "jail breaks" in your comments, and update them - so that future readers are not confused.